src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -366,6 +366,23 @@ public OperationQueue<TOperation> Stop()
             }
         }
 
+        private static SocketAsyncContext _closedAsyncContext;
```

Comment:
I would say "shouldn't this field be down with the other fields?" but it looks like they're sort of sprinkled about in this particular class.

But it probably should be `s_closedAsyncContext` :smile: